README.md

```diff
@@ -25,15 +25,43 @@ See `api.md` for an in-depth description of Supervisor and how it's used.
 1. Install supervisor (or add to your project's `setup.py`)
    - `pip install agrc-supervisor`
 1. In your script's entry point code (usually `main.py`), as early as possible and generally before any arg parsing:
-   - Instantiate a `Supervisor` object
+   - Instantiate a `Supervisor` object, passing an existing logger to log any errors.
```

Comment:
**nitpick**: this is optional
```suggestion
   - Instantiate a `Supervisor` object, optionally passing an existing logger to log any caught errors.
```

Reply:
Yeap, the docs and example have been updated to reflect that.